src/verifier/compare_outputs.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+if [ $# -eq 0 ]; then
+    echo "Missing argument for name of existing CSV file from verified_output to check against."
+    exit $1
```

Comment:
A fuller usage message would be nice.  i.e. with `usage: verifier BASE_FILE [COMPARISON_FILE]` and a note that the arguments should be relative to `./verified_output` and `./data/output` respectively.

Also this `exit $1` should be `exit 1`, right?